Formula/jupyterlab.rb

```diff
@@ -354,6 +354,7 @@ def install
     # gather packages to link based on options
     linked = %w[jupyter-core jupyter-client nbformat ipykernel jupyter-console nbconvert notebook]
     dependencies = resources.map(&:name).to_set - linked
+    on_linux { dependencies -= appnope }
```

Comment:
May need to do:
```suggestion
    on_linux { dependencies -= [resource("appnope")] }
```
Or some other variation.